src/main/webapp/app/exercises/quiz/manage/statistics/quiz-statistic/quiz-statistic.component.ts

```diff
@@ -136,6 +106,15 @@ export interface DataSetProvider {
     getParticipants(): number;
 }
 
+export function calculateTickMax(datasetProvider: DataSetProvider) {
+    const data = datasetProvider.getDataSets().map((dataset) => {
+        return dataset.data;
+    });
+    const merged = [].concat.apply([], data);
+    const max = Math.max(...merged);
```

Comment:
what are you doing here, can't we jsut do `Math.max(...data)` (do we need the `[].concat.apply([],data)`)

Reply:
should then be `Math.max([...data])` I think.

Reply:
Hi the type of `data` is 
![image](https://user-images.githubusercontent.com/29718932/91536595-3de19380-e915-11ea-8e38-8d272255212d.png) so we need to flatten it first one leve

Reply:
I modified the code a little to make it more clearer